.github/workflows/AutoMerger.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow syncs automatically merges pull requests under
+# certain conditions in Project Mu repos.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+# For more information, see:
+# https://github.com/pascalgn/automerge-action
+
+name: Auto Merge Pull Request Workflow
+
+on:
+  workflow_call:
+
+jobs:
+  auto_merge:
+    name: Merge
+    runs-on: ubuntu-latest
+
+    # The action cannot take multiple authors right now, so call with each author
+    # separately.
+    strategy:
+      matrix:
+        author: [ "dependabot[bot]", "uefibot" ]
+
+    steps:
+      - name: Auto Merge
+        id: auto_merge
+        uses: pascalgn/automerge-action@v0.15.5

```

Comment:
looking towards the future: is there any concern about relying on an external developer like this?

Reply:
Not significantly given permissions we've granted. I do plan to go through and see if we can reduce permissions further for the actions we're using.